cmd/brew-services.rb

```diff
@@ -270,30 +255,21 @@ def cleanup
     end
 
     # Stop if loaded, then start again
-    def restart
-      services = target_services
-      @act_on_all_services = false
-
-      services.each do |service|
-        @target_services = [service]
-        @single_service = service
-
-        stop if service.loaded?
-        start
+    def restart(target)
+      Array(target).each do |service|
```

Comment:
`.to_a` and `Array()` are a bit different. `.to_a` relies on the internal implementation of that method and if in fact missing by default. While `Array()` will try calling `.to_a` on the passed in object, but will simply wrap it in `[]` in case it's missing. I prefer to think of it as a `.to_a` with a fallback. And since `Service` doesn't have `.to_a` implemented, it does the trick.